ArduCopter/mode_althold.cpp

```diff
@@ -75,8 +73,7 @@ void ModeAltHold::run()
         target_climb_rate = get_avoidance_adjusted_climbrate(target_climb_rate);
 
         // set position controller targets
-        pos_control->set_alt_target_from_climb_rate_ff(target_climb_rate, G_Dt, false);
-        pos_control->add_takeoff_climb_rate(takeoff_climb_rate, G_Dt);
+        pos_control->set_alt_target_from_climb_rate(target_climb_rate + takeoff_climb_rate, false);
```

Comment:
I would have expected "set_alt_target_from_climb_rate" to be renamed to, "set_pos_target_z_from_climb_rate"